code/game/gamemodes/miniantags/guardian/guardian.dm

```diff
@@ -271,19 +277,20 @@
 	if(candidates.len)
 		theghost = pick(candidates)
 		spawn_guardian(user, theghost.key)
+		desc += "\n[used_message]"
```

Comment:
Would it not be more accurate to add a to_chat (with span class) in the examine proc to show whether it is used instead?